scripts/export-git-env-vars.sh

```diff
@@ -22,7 +22,7 @@ function find_most_recent_numeric_tag() {
 export GITHASH=$(git rev-parse --short=7 HEAD)
 export THIS_COMMIT_TAG=$(git tag --points-at HEAD)
 
-if [[ -n "$BRANCH_NAME" ]]; then
```

Comment:
Without this change in `GNU bash, version 5.2.21(1)-release (x86_64-pc-linux-gnu)` I got errors when these vars are not set